cove_360/lib/threesixtygiving.py

```diff
@@ -396,6 +396,18 @@ def process(self, grant, grant_flat, path_prefix):
         self.message = "Users may find that the data is less useful as they are unable to discover more about the grants. Consider including a more detailed description if you have one."
 
 
+class TitleLength(AdditionalTest):
+    def process(self, grant, grant_flat, path_prefix):
+        title = grant.get("title", '')
```

Comment:
How does this handle the case of (invalid) data that doesn't have a title field?